snapcraft.yaml

```diff
@@ -815,6 +815,7 @@ parts:
       - --disable-parallels
       - --disable-qed
       - --disable-slirp
+      - --disable-tcg
```

Comment:
I understand this is needed because we use qemu to setup uefi keys during snap build. Which doesn't have kvm.available.

Reply:
Ah, that could explain why `qemu-ovmf-secureboot` no longer builds with this, too bad!

Reply:
That's the one. @mihalicyn has an item for this but depends on secure boot keys being available as a package in ubuntu.